src/encoded/static/components/experiment_series.js

```diff
@@ -520,10 +522,16 @@ class ExperimentSeriesComponent extends React.Component {
         if (Object.keys(viewableDatasets).length > 0) {
             // Add the "Add all to cart" button and internal tags from all related datasets.
             const experimentIds = Object.values(viewableDatasets).map(experiment => experiment['@id']);
```

Comment:
I missed this the first time, and this does work in this case by accident, but the `window` global is only available in the browser, so in theory when this code runs on the server it crashes. For these cases you should use the `location_href` property from `<App>` context. In this case it works because viewableDatasets doesn’t have a value on the server; only in the browser.